readme.md

```diff
@@ -222,6 +222,7 @@
 - [Plus AI for Google Slides](https://www.plusdocs.com/plus-ai-for-google-slides) - Create AI-powered presentations in Google Slides.
 - [Scikit-LLM](https://github.com/iryna-kondr/scikit-llm) - Integrate ChatGPT capabilities into scikit-learn.
 - [ChatGPT-MD](https://github.com/bramses/chatgpt-md) - ChatGPT integration with Obsidian note-taking software.
+- [I Don't Care About Commit Message (VS Code Extension)](https://github.com/mefengl/vscode-i-dont-care-about-commit-message) - Yet another AI git commit plugin, but without the need for manual confirmation.
```

Comment:
> Yet another AI git commit plugin

This is not useful text.

Reply:
> (VS Code Extension)

Would be better to have this in the description.

Reply:
Thank you for your advice. I hope things will be better now.